3rd-party-licenses.md

```diff
@@ -0,0 +1,147 @@
+# License overview of included 3rd party libraries
+
+Shopsys Framework is licensed under the terms of the [Shopsys Community License](./LICENSE).
+
+Shopsys Framework also uses some third-party components and images
+which are licensed under their own respective licenses.
+
+## Components used by Shopsys Framework
+### Symfony Framework and Symfony Components
+License: MIT  
+https://symfony.com/
+
+### Elasticsearch
+License: Apache License 2.0  
+https://github.com/elastic/elasticsearch-php/blob/master/LICENSE
+
+### Grunt: The JavaScript Task Runner
+License: MIT  
+https://github.com/gruntjs/grunt
+
+### Phing
+License: LGPL-3.0-only  
+https://github.com/phingofficial/phing/blob/master/LICENSE
+
+### Nette Foundation tools 
+License: BSD-2-Clause or GPL-2.0 or GPL-3.0  
+https://nette.org/
+
+### PHP_CodeSniffer
+License: BSD-3-Clause  
+https://github.com/squizlabs/PHP_CodeSniffer
+
+### PHP Parallel Lint
+License: BSD-2-Clause  
+https://github.com/JakubOnderka/PHP-Parallel-Lint/blob/master/LICENSE
+
+### ESLint
+License: MIT  
+https://github.com/eslint/eslint/blob/master/LICENSE
+
+### jQuery
+License: MIT or GPL Version 2  
+http://jquery.org/license
+
+### selectize.js
```

Comment:
I would rather have a individual section for the included JS libraries and where to find them. This might not be evident and is quite important for further maintenance of this list.

Reply:
Im not sure if this is the scope of this US. I do not think we have ever written detailed instructions on how to replicate a procedure. For example, when we created docker-compose files or kubernetess files, have there been any instructions on how to add a new container or change the image? But, I'm trying to write procedures for everything, even if others do not :( . That's why I added a new section to our internal document :). Please review it.